File: core-sample/src/siminov/core/sample/model/LiquorShopMapping.java
package siminov.core.sample.model;

import siminov.core.database.Database;

public class LiquorShopMapping extends Database {

	//Table Name
	transient public static final String TABLE_NAME = "LIQUOR_SHOP_MAPPING";
	
	//Variable Names
	private Liquor liquor = null;
	private Shop shop = null;
	
	
	//Method Names
	
	public Liquor getLiquor() {
		return this.liquor;
	}
	
	public void setLiquor(Liquor liquor) {
		this.liquor = liquor;
	}
	
	public Shop getShop() {
		return this.shop;
	}
	
	public void setShop(Shop shop) {
		this.shop = shop;
	}
}

File: Core-Sample/iOS/Siminov/Core/Sample/Model/LiquorShopMapping.m
//
//  LiquorShopMapping.m
//  core-sample
//
//  Created by user on 13/04/15.
//  Copyright (c) 2015 Siminov. All rights reserved.
//

#import "LiquorShopMapping.h"

@implementation LiquorShopMapping

-(Liquor *)getLiquor {
    return liquor;
}

-(void)setLiquor:(Liquor *)liq {
    liquor = liq;
}


-(Shop *)getShop {
    return shop;
}

-(void)setShop:(Shop *)shp {
    shop = shp;
}


@end
